src/photometry.cc

```diff
@@ -160,7 +160,7 @@ lsst::afw::table::BaseCatalog photometer(
                 // Bad PSF: not even a blend.
                 continue;
             }
-            box.shift(lsst::geom::Extent2D(point) - 0.5*box.getDimensions());  // put the center on the point
+            box.shift(lsst::geom::Extent2D(point));  // put the center on the point
```

Comment:
In the commit comment, you mentioned that the shifting is 'unnecessary', but it is more than unnecessary, surely, it is erroneous?